# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kdepimlibs/sasl2.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- kdepimlibs-4.0.1/kldap/ldapconnection.cpp.orig	2008-02-25 10:31:59.000000000 +0100
+++ kdepimlibs-4.0.1/kldap/ldapconnection.cpp	2008-02-25 10:35:40.000000000 +0100
@@ -27,7 +27,7 @@
 #include <kdebug.h>
 
 #ifdef SASL2_FOUND
-#include <sasl/sasl.h>
+#include <sasl2/sasl.h>
 static sasl_callback_t callbacks[] = {
   { SASL_CB_ECHOPROMPT, NULL, NULL },
   { SASL_CB_NOECHOPROMPT, NULL, NULL },
--- kdepimlibs-4.0.1/kldap/ldapoperation.cpp.orig	2008-02-25 10:31:59.000000000 +0100
+++ kdepimlibs-4.0.1/kldap/ldapoperation.cpp	2008-02-25 10:35:47.000000000 +0100
@@ -31,7 +31,7 @@
 #endif
 
 #ifdef SASL2_FOUND
-#include <sasl/sasl.h>
+#include <sasl2/sasl.h>
 #endif
 
 #ifdef LDAP_FOUND
--- kdepimlibs-4.0.1/kioslave/pop3/pop3.cpp.orig	2008-02-25 10:31:59.000000000 +0100
+++ kdepimlibs-4.0.1/kioslave/pop3/pop3.cpp	2008-02-25 10:35:54.000000000 +0100
@@ -40,7 +40,7 @@
 
 #ifdef HAVE_LIBSASL2
 extern "C" {
-#include <sasl/sasl.h>
+#include <sasl2/sasl.h>
 }
 #endif
 
--- kdepimlibs-4.0.1/kioslave/smtp/command.h.orig	2008-02-25 10:31:59.000000000 +0100
+++ kdepimlibs-4.0.1/kioslave/smtp/command.h	2008-02-25 10:36:01.000000000 +0100
@@ -37,7 +37,7 @@
 
 #ifdef HAVE_LIBSASL2
 extern "C" {
-#include <sasl/sasl.h>
+#include <sasl2/sasl.h>
 }
 #endif
 
--- kdepimlibs-4.0.1/kioslave/smtp/smtp.cpp.orig	2008-02-25 10:31:59.000000000 +0100
+++ kdepimlibs-4.0.1/kioslave/smtp/smtp.cpp	2008-02-25 10:36:07.000000000 +0100
@@ -32,7 +32,7 @@
 
 #ifdef HAVE_LIBSASL2
 extern "C" {
-#include <sasl/sasl.h>
+#include <sasl2/sasl.h>
 }
 #endif
 
--- kdepimlibs-4.0.1/kioslave/imap4/imapparser.cpp.orig	2008-02-25 10:31:59.000000000 +0100
+++ kdepimlibs-4.0.1/kioslave/imap4/imapparser.cpp	2008-02-25 10:36:12.000000000 +0100
@@ -36,7 +36,7 @@
 
 #ifdef HAVE_LIBSASL2
 extern "C" {
-#include <sasl/sasl.h>
+#include <sasl2/sasl.h>
 }
 #endif
 
--- kdepimlibs-4.0.1/kioslave/imap4/imap4.cpp.orig	2008-02-25 10:31:59.000000000 +0100
+++ kdepimlibs-4.0.1/kioslave/imap4/imap4.cpp	2008-02-25 10:36:15.000000000 +0100
@@ -70,7 +70,7 @@
 
 #ifdef HAVE_LIBSASL2
 extern "C" {
-#include <sasl/sasl.h>
+#include <sasl2/sasl.h>
 }
 #endif
 
--- kdepimlibs-4.0.1/kioslave/sieve/sieve.cpp.orig	2008-02-25 10:31:59.000000000 +0100
+++ kdepimlibs-4.0.1/kioslave/sieve/sieve.cpp	2008-02-25 10:36:20.000000000 +0100
@@ -27,7 +27,7 @@
 #include "sieve.h"
 
 extern "C" {
-#include <sasl/sasl.h>
+#include <sasl2/sasl.h>
 }
 
 #include <kdemacros.h>
